src/main/java/tempest/ui/components/BackButton.java

```diff
@@ -19,15 +17,4 @@ public BackButton(GUIManager manager) {
         this.setFocusable(false);
         this.addActionListener(e -> manager.swapToPrevCard());
     }
-
-    /**
-     * Navigation button that sends the user back to the previous chart.
-     * 
-     * @param manager The instance of manager passed into the page class
-     */
-    public BackButton(ViewManager<Chart> manager) {
-        super("Back");
```

Comment:
Why is the name of the button not set in the new implementation?

Reply:
It is in the method above _(had to extend the code to see it)_

```java
public BackButton(GUIManager manager) {
    super("Back");
    this.setFocusable(false);
    this.addActionListener(e -> manager.swapToPrevCard());
}
```

That ViewManager<Chart> one wasn't used by anything